docs/docs/node-model.md

```diff
@@ -1,14 +1,14 @@
 ---
 title: Node Model
-description: Documentation explaining the model of nodes in Gatsby's GraphQL data layer
+description: Gatsby's ग्राफक्यूएल डेटा लेयरमधील नोड्सचे मॉडेल स्पष्ट करणारे दस्तऐवजीकरण
```

Comment:
ग्राफक्यूएल  it should de as it is in English.

Reply:
yes, please dont translate GraphQL